src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheckTest.java

```diff
@@ -98,9 +98,9 @@ public void testNullCustomMessage()
 
         checkConfig.addProperty("message", null);
```

Comment:
How do we replicate this in config comment?

Reply:
https://github.com/checkstyle/checkstyle/issues/10527